libs/langchain/langchain/agents/agent_toolkits/__init__.py

```diff
@@ -81,4 +83,5 @@
     "create_vectorstore_router_agent",
     "create_xorbits_agent",
     "create_conversational_retrieval_agent",
+    "SlackToolkit",
```

Comment:
delete this line please since there's a duplicate one on line 64

Reply:
deleted